ksqldb-engine/src/test/java/io/confluent/ksql/services/SandboxedSchemaRegistryClientTest.java

```diff
@@ -196,15 +197,21 @@ public void shouldGetSchemaById() throws Exception {
 
     @Test
     public void shouldGetId() throws Exception {
+      // Given:
+      when(delegate.getId(anyString(), any(ParsedSchema.class)))
```

Comment:
Do we need case where `delegate` throws `SC_NOT_FOUND` but mock client returns id?